gateway-spi/src/main/java/org/apache/knox/gateway/services/security/token/TokenMetadata.java

```diff
@@ -142,6 +139,32 @@ public Instant getLastUsedAt() {
     return lastUsedAt == null ? null : Instant.parse(lastUsedAt);
   }
 
+  public void setType(String type) {
```

Comment:
Can we use just enum Type for simplicity?

Reply:
Ack.